net-p2p/go-ipfs/go-ipfs-0.11.0.ebuild

```diff
@@ -1726,7 +1955,8 @@ src_test() {
 src_install() {
 	dobin ipfs
 	dobin ipfswatch
-	newbashcomp misc/completion/ipfs-completion.bash ipfs
+	./ipfs commands completion bash > ipfs-completion.bash
```

Comment:
|| die on external commands which may fail

Reply:
Does this even make sense? It just creates the file in the build directory but it never installs it.

I believe the `newbashcomp` should stay in place, and the generator should go into the build phase?

Reply:
Doesn't it install it on the next line? But yeah, it could/should maybe go into src_compile.

Reply:
Ah okay, on the full view I can see the next line - it was missing on the review comments...

Maybe this `newbashcomp` line shouldn't change at all, and instead the compile phase should just create the file at the proper location `misc/completion/...`.

Reply:
It installs it after see the next line:
```
    ./ipfs commands completion bash > ipfs-completion.bash
    newbashcomp ipfs-completion.bash ipfs
```

Reply:
> Doesn't it install it on the next line? But yeah, it could/should maybe go into src_compile.

The problem is that the generator requires ipfs binary to be compiled, so if it's in src_compile, as the binary is still not compiled it will use the generator for the previous version, of it's the first time it's emerged it will fail

Reply:
I think it would work if you called default first then put this line after, right? Not that I'm super bothered about this, it is a valid point

Reply:
Oh yes, right, it was obvious sry

Reply:
> as the binary is still not compiled it will use the generator for the previous version

That's the same situation with the current approach: `src_install()` doesn't actually install to the search path, it installs to some sort of "chroot" directory (the image path). You are probably using `./ipfs` for that very reason. And this means, the binary is available in the build path - just like it would be at the end of the `src_compile()` function - or did I miss something?